src/main/java/frc/trigon/robot/subsystems/arm/ArmConstants.java

```diff
@@ -0,0 +1,60 @@
+package frc.trigon.robot.subsystems.arm;
+
+import edu.wpi.first.math.geometry.Rotation2d;
+import edu.wpi.first.math.trajectory.TrapezoidProfile;
+import edu.wpi.first.wpilibj.smartdashboard.Mechanism2d;
+import edu.wpi.first.wpilibj.smartdashboard.MechanismLigament2d;
+import edu.wpi.first.wpilibj.smartdashboard.MechanismRoot2d;
+import edu.wpi.first.wpilibj.util.Color;
+import edu.wpi.first.wpilibj.util.Color8Bit;
+
+public class ArmConstants {
+    private static final double
+            MAX_ANGLE_VELOCITY = 150,
+            MAX_ELEVATOR_VELOCITY = 2,
+            MAX_ANGLE_ACCELERATION = 190,
+            MAX_ELEVATOR_ACCELERATION = 3;
+
+    static final TrapezoidProfile.Constraints
+            ANGLE_CONSTRAINTS = new TrapezoidProfile.Constraints(MAX_ANGLE_VELOCITY, MAX_ANGLE_ACCELERATION),
+            ELEVATOR_CONSTRAINTS = new TrapezoidProfile.Constraints(MAX_ELEVATOR_VELOCITY, MAX_ELEVATOR_ACCELERATION);
+    static final double
+            ANGLE_TOLERANCE_DEGREES = 0.5,
+            ELEVATOR_TOLERANCE_METERS = 0.02;
+    public static final double
+            ARM_MECHANISM_WIDTH_METERS = 3,
+            ARM_MECHANISM_HEIGHT_METERS = 3,
+            ARM_LIGAMENT_LENGTH_METERS = 0.64,
+            ARM_LIGAMENT_ANGLE_DEGREES = 0,
+            RETRACTED_ARM_LENGTH_METERS = 0.64;
```

Comment:
The retracted arm length is not similar to the mechanism width therefore you should type the type for it again.